node/src/Pos/Wallet/Web/ClientTypes/Functions.hs

```diff
@@ -54,7 +54,7 @@ cIdToAddress :: CId w -> Either Text Address
 cIdToAddress (CId (CHash h)) = decodeTextAddress h
 
 encToCId :: EncryptedSecretKey -> CId w
-encToCId = addressToCId . makePubKeyAddress . encToPublic
+encToCId = addressToCId . makePubKeyAddress undefined . encToPublic
```

Comment:
Do you mean `IsBootstrapEraAddr True`? That's what I am going to do, also will add a TODO comment.